packages/runtime/src/components/builtin/Image/Image.tsx

```diff
@@ -1,5 +1,6 @@
 import { useState, useEffect, Ref, forwardRef } from 'react'
 import NextImage from 'next/image'
+import _NextLegacyImage from 'next/legacy/image'
```

Comment:
To avoid actually importing it and including it in the output you can explicitly do `import typeof NextLegacyImageType from 'next/legacy/image'`